### PR TITLE
Initial Node e2e tests

### PR DESCRIPTION
**Pull request by pwittrock**

### CODE REVIEW

File: hack/verify-flags/known-flags.txt [hunk 1]
```diff
@@ -10,10 +10,10 @@ algorithm-provider
 all-namespaces
 allocate-node-cidrs
 allow-privileged
+api-server-address
```
Comment (vishh) [inline hack/verify-flags/known-flags.txt]: Should this test framework reside in kubernetes/contrib instead?

Comment (vishh) [inline hack/verify-flags/known-flags.txt]: nvm. It should be useful to most kube users. So its ok to be here. 
I just find it weird that we group test-only and production flags in a single file. 

This file should probably be per-component.

**Comment by vishh** [comment]
LGTM

**Comment by pwittrock** [comment]
@k8s-bot test this

**Comment by pwittrock** [comment]
@k8s-bot test this

Looks like the failed test was a timeout in /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/daemon_set.go  Should be unrelated.

**Comment by pwittrock** [comment]
@k8s-bot test this

Looks like the test timed out and panicked.

**Pull request merged**